toolbox/bin/variables.sh

```diff
@@ -58,7 +58,9 @@ fi
 # populate current layer info
 LAYERS_DIR="$WORKING_DIR/layers"
 LAYERS=''
-if [ -d "$LAYERS_DIR" ]; then
+if [ ! -d "$LAYERS_DIR" ]; then
```

Comment:
This feels weird to do on a repo that doesn't have layers. Should we add a comment to explain why this is here?